types/src/api/response.rs

```diff
@@ -302,6 +302,20 @@ impl OutputMetadataResponse {
     }
 }
 
+/// Response of GET /api/core/v2/outputs/{output_id}.
+/// Returns an output and its metadata as JSON or raw bytes.
```

Comment:
It's like this, right?
```suggestion
/// Returns an output and its metadata as JSON or only the output as raw bytes.
```